public/js/reports/c3Charts.service.js

```diff
@@ -228,6 +227,14 @@ angular.module('app.reports').service('c3Charts', function (notify, gettextCatal
                         multiline: false,
                         rotate: 45
                     }
+                },
+                y: {
+                    tick: {
+                        format: function (d) {
+                            const format = report.properties.ykeys[0].format;
```

Comment:
You can have several columns in `ykeys`, each with their own format. Here you are using the 1st column format for all columns.
Also, you should not call numeral's `.format()` if no format is defined